cost/aws/elb/clb_unused/aws_delete_unused_clb.pt

```diff
@@ -471,8 +473,13 @@ script "js_clbs_cost_mapping", type:"javascript" do
         })
       }
     })
-    var total_savings=cur + ' '+formatNumber(total.toFixed(3), separator);
-    message=message="The total estimated monthly savings are "+total_savings;
+    if(count){
+      var total_savings=cur + ' '+formatNumber((Math.round(total * 100) / 100), separator);
+      message=message="The total estimated monthly savings are "+total_savings;
+    }
+    else{
+      message="The data received from Optima is not enough to calculate the savings";
```

Comment:
I think it might be better to phrase this as something like:

> The Flexera Optima system does not yet have any data to calculate savings for this resource

Reply:
modified the policy.